polaris-react/src/components/Button/Button.scss

```diff
@@ -424,8 +424,8 @@
     position: relative;
     border-color: currentColor;
     // stylelint-disable selector-max-class
-    box-shadow: 0 0 0 var(--p-border-width-1) currentColor;
```

Comment:
I didn't feel like `border-width` should be used for `blur-radius` for `box-shadow`.

Reply:
Agreed, we should be leveraging a `shadows` token scale here. I believe we still need to establish that scale and a mechanism for defining the color. So perhaps this is a special case we have to leave for now 🤷  Curious what other folks think.